src/diffpy/labpdfproc/labpdfprocapp.py

```diff
@@ -30,14 +30,15 @@ def get_args():
     p.add_argument(
         "-o",
         "--output-directory",
-        help="the name of the output directory. If it doesn't exist it "
-        "will be created.  Not currently implemented",
+        help="the name of the output directory. If not specified "
+        "then corrected files will be written to the current directory."
+        "If the specified directory doesn't exist it will be created.",
         default=None,
     )
     p.add_argument(
         "-x",
         "--xtype",
-        help=f"the quantity on the independnt variable axis. allowed "
+        help=f"the quantity on the independent variable axis. allowed "
```

Comment:
Let's capitalize all the first letters of all the help statements so it is better grammar.